### PR TITLE
K8s Job Command Clean

### PR DESCRIPTION
**Pull request by njgibbon**
Hello,

* Updates the `job-*.yamls` for consistency purposes and updated to use  `run --targets` as the prior statements in-use have been deprecated.

ItAintMuch

Thank you for Kube-bench.

### CODE REVIEW

File: job-iks.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
       containers:
         - name: kube-bench
           image: aquasec/kube-bench:latest
-          command: ["kube-bench", "--version", "1.13", "node"]
+          command: ["kube-bench", "run", "--targets", "node", "--version", "1.13"]
```
Comment (yoavrotems) [inline job-iks.yaml:13]: We should also update the version here to 1.20 🤙

**Review by yoavrotems — CHANGES_REQUESTED** [review]

**Comment by yoavrotems** [comment]
Beside that command everything looks good and a nice honest work !  😉

File: job-iks.yaml (hunk 1 above)
Comment (njgibbon) [reply job-iks.yaml]: Hey - yes, makes sense thank you :) Just updated.

**Review by yoavrotems — APPROVED** [review]

**Comment by yoavrotems** [comment]
LGTM great! :)

**Pull request merged**